### ISSUE TITLE
replacement transaction underpriced

### ISSUE THREAD

**Issue by farhankhwaja** [kvhnuke/etherwallet#1050]
I tried deploying a contract with the below addresses:

- 0x76E0bb92b6F8d431Af2Fe1BAfd304EB832241619
- 0xb1aa0ac057639e534a89b095853befc4d6d264cf

but none of the transaction went through, and when I ma trying to go another transaction from the same address, the app is showing a notification stating `replacement transaction underpriced`.

How can i make my accounts functional again?

**Comment by ernaneluis** [comment]
I'm facing same problem. maybe is related to this issue: https://github.com/ethereum/go-ethereum/issues/15186

**Comment by tayvano** [comment]
Can either of you please share the code you are attempting to deploy so Incan recreate & test fix. I will do my best to solve as soon as possible. 

Thank you.

**Comment by karalabe** [comment]
A new transaction must be at least 10% more expensive gas-price wise than the old one to be accepted by Geth nodes. This is to prevent DOS/SPAM attacks where you inject transactions with a gas price of [1, 2, 3, 4, ..., infinity], constatnly having the network propagate the data.

**Comment by ernaneluis** [comment]
Well I think MEW is not estimating good enough on the ropsten chain. I made it work by setting up my gas limit until it worked by no more _out of gas_ error 

MEW TX:
```
Gas Limit: 132522 
Gas Used By Txn: 132521 
Gas Price: 0.000000021 Ether (21 Gwei) 
Actual Tx Cost/Fee: 0.002782941 Ether ($0.000000) 
Cumulative Gas Used: 1511518
```

My TX:
```
Gas Limit: 4712388 
Gas Used By Txn: 161931 
Gas Price: 0.0000001 Ether (100 Gwei) 
Actual Tx Cost/Fee: 0.0161931 Ether ($0.000000) 
Cumulative Gas Used: 161931
```

**Comment by karalabe** [comment]
Yeah, gas estimation has some issues starting from Byzantium. Solidity
converts all CALL failures into reverts now, which causes internal OOG
errors to silently disappear..

**Comment by tayvano** [comment]
Is there anything that we can do on MEW's end to resolve this or improve the error messaging?

**Comment by karalabe** [comment]
Gas estimation will be fixed in today's release. I can expand the other error a bit too to contain the min next gas price for replacement.

**Comment by farhankhwaja** [comment]
I solved by making a new transaction with increased gasLimit.

@tayvano I was using MEW (on Ropsten) and did a transaction using my walletAddress. I didn't get a transaction Hash back but the transaction got pushed onto the blockchain (i could see the transaction with my wallet-address on etherscan). And, when I tried doing another transaction with lower gasLimit then the previous one, it started failing.
